TestingLowerBounds/Renyi.lean

```diff
@@ -53,152 +53,165 @@ lemma exp_mul_llr' [SigmaFinite μ] [SigmaFinite ν] (hμν : μ ≪ ν) :
 
 /-- Rényi divergence of order `a`.-/
 noncomputable def renyiDiv (a : ℝ) (μ ν : Measure α) : EReal :=
-  if a = 0 then - log (ν {x | 0 < (∂μ/∂ν) x}).toReal
-  else if a = 1 then kl μ ν
+  if a = 1 then kl μ ν
   else if hellingerDiv a μ ν ≠ ⊤
-    then (a - 1)⁻¹ * log (1 + (a - 1) * (hellingerDiv a μ ν).toReal)
+    then (a - 1)⁻¹ * log ((ν Set.univ).toReal + (a - 1) * (hellingerDiv a μ ν).toReal)
```

Comment:
Is it ok to write in a less formal way in the dosctring, in general? Or should I stick with lean notation?
For example `(a - 1)⁻¹ * log (ν(α) + (a - 1) * Hₐ(μ, ν))` instead of `(a - 1)⁻¹ * log ((ν Set.univ).toReal + (a - 1) * (hellingerDiv a μ ν).toReal)`. It seems to make the docs more readable, but it doesn't stick completely with mathlib notation.

Reply:
Yes that's fine.